backend/development.ini.sample

```diff
@@ -313,14 +313,14 @@ email.notification.activated = False
 
 
 ### SMTP ###
-email.notification.smtp.server = your_smtp_server
-email.notification.smtp.port = 25
-email.notification.smtp.user = your_smtp_user
-email.notification.smtp.password = your_smtp_password
+; email.notification.smtp.server = your_smtp_server
+; email.notification.smtp.port = 25
+; email.notification.smtp.user = your_smtp_user
+; email.notification.smtp.password = your_smtp_password
 # active implicit ssl if you are using implicit smtp with encryption port like 465
 # by default, tracim will try to use explicit smtp encryption using starttls, and unencrypted
 # connection as fallback.
-email.notification.smtp.use_implicit_ssl = false
+; email.notification.smtp.use_implicit_ssl = false
 
 ### Headers ###
 email.notification.from.default_label = Tracim Notifications
```

Comment:
No default value here ?

Reply:
Adding information: i speak about `email.notification.from.default_label = Tracim Notifications` (adding this comment after speak with inkhey)